compiler/rustc_feature/src/removed.rs

```diff
@@ -163,6 +163,9 @@ declare_features! (
     (removed, quad_precision_float, "1.0.0", None, None, None),
     (removed, quote, "1.33.0", Some(29601), None, None),
     (removed, reflect, "1.0.0", Some(27749), None, None),
+    /// Allows using the `#[register_attr]` attribute.
+    (removed, register_attr, "CURRENT_RUSTC_VERSION", Some(66080), None,
```

Comment:
Could you add a "FIXME" or keep #66080 open until the next version is released?

Reply:
I'm not sure what the intention is to keep the tracking issue open as the feature will be completely removed. Could you clarify?

Reply:
I forgot to mention why. This is to remind us to change `CURRENT_RUSTC_VERSION` to a fixed version.

Reply:
How does it help? According to https://github.com/rust-lang/rust/pull/100591 and https://forge.rust-lang.org/release/process.html#master-bootstrap-update-t-2-day-tuesday, `x.py` will replace it with an actual version number when preparing a new release, and I don't see any relation of the tracking issues at all. But it's kinda new stuff and I might be missing something, do you have any references for that?

Reply:
Sorry. I did not fully understand the release process. Thanks for the explanation. It looks good.

Reply:
@JohnTitor your understanding is correct.